docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -662,6 +663,10 @@ _**default:**_ `application/atom+xml application/javascript application/x-javasc
 Sets the number of [worker processes](http://nginx.org/en/docs/ngx_core_module.html#worker_processes).
 The default of "auto" means number of available CPU cores.
 
+## worker-rlimit-core
+
+Sets the limit on the [largest size of a core file](http://nginx.org/en/docs/ngx_core_module.html#worker_rlimit_core).
```

Comment:
Can you please add what is the expected format? like "100m" or "100kb" or 100 and this is converted to bytes